os/arch/arm/src/armv7-r/arm_saveusercontext.S

```diff
@@ -140,6 +140,16 @@ up_saveusercontext:
 	vmrs	r2, fpscr				/* Fetch the FPCSR */
 	str		r2, [r1], #4			/* Save the floating point control and status register */
 #endif
+	/* Save the MPU register information into the context structure */
+
+#if defined(CONFIG_BUILD_PROTECTED)
+	sub	sp, #4
```

Comment:
Same as above.

Reply:
@satheesh-ks please check this

Reply:
Hi All,
This patch is simple revert of commit 2dbd158de140bf14ec4ac471ee9640542708f8bc.
So in the original patch it might have in "-" section which might be added as "+" section in my reversion